packages/api/provision/src/route/servers.rs

```diff
@@ -53,7 +53,7 @@ pub async fn info(
 		server_id: server.server_id,
 		datacenter_id: server.datacenter_id,
 		cluster_id: datacenter.cluster_id,
-		vlan_ip: unwrap_ref!(server.vlan_ip, "server should have vlan ip by now").to_string(),
-		public_ip: unwrap_ref!(server.public_ip, "server should have public ip by now").to_string(),
+		lan_ip: unwrap_ref!(server.lan_ip, "server should have lan hostname by now").to_string(),
```

Comment:
i dont think this will cause any issues but olders servers that reboot and run the rivet_fetch_info.sh script again will fail because vlan_ip is gone

Reply:
added deprecated fields